src/ProtoRoute.hs

```diff
@@ -8,10 +8,14 @@ import ProtoRoute.Message (MsgName (..), FieldName (..), TValue (..),
                            FieldValue (..), constructProtoMsg)
 
 main :: IO ()
-main = runGhci $ constructProtoMsg sreq [(query, someText), (unknown, none)]
+main = runGhci $ constructProtoMsg msg [(field1, value1)
+                                      , (field2, value2)
+                                      , (unknown, none)]
   where
-    sreq  = MN "SearchRequest"
-    query = FN "'query"
-    someText = FText (Req $ pack "someText")
-    unknown = FN "unknownFields"
-    none = FText (Opt $ Nothing)
+    msg = MN "SearchRequest"
+    field1 = FN "query"
+    value1 = FText (Req $ pack "someText")
+    field2 = FN "browser"
+    value2 = FText (Opt (Just $ pack "Safari"))
+    unknown = FN "_unknownFields"
+    none = FText (Rep $ [])
```

Comment:
`$` here is redundant. You can run your code through `hlint` to cleanup it a little bit.